docs/design/kata-nydus-design.md

```diff
@@ -0,0 +1,91 @@
+# Background
+
+[Research](https://www.usenix.org/conference/fast16/technical-sessions/presentation/harter) shows that time to take for pull operation accounts for 76% of container startup time but only 6.4% of that data is read. So if we can get data on demand (lazy load), it will speed up the container start. [Nydus](https://github.com/dragonflyoss/image-service) is a project which build image with new format and can get data on demand when container start. Nydus has supported runc containers.
+
+The following benchmarking result shows the performance improvement compared with the OCI image for the container cold startup elapsed time on containerd. As the OCI image size increases, the container startup time of using Nydus image remains very short.
+
+![nydus-profile](arch-images/perf.jpeg)
+
+## Proposal - Bring lazyload ability to Kata Containers
+
+Nydusd is a fuse/virtiofs daemon which is provided by nydus project and it supports Passthroughfs and [RAFS](https://github.com/dragonflyoss/image-service/blob/master/docs/nydus-design.md) (Registry Acceleration File System) natively, so in Kata Containers, we can use nydusd in place of virtiofsd and mount nydus image to guest in the meanwhile.
+
+The process of creating/starting Kata Containers with virtiofsd,
+
+1. when creating sandbox, the Kata Containers shim will launch virtiofsd before VM starts, so VM can share directories with host.
```

Comment:
Could you link the word "shim" to https://github.com/kata-containers/kata-containers/blob/main/docs/design/architecture/README.md#runtime here.

Reply:
agree, it would be better to provide nydus  architecture document

Reply:
I add `nydusd.md` which is copied from (https://github.com/dragonflyoss/image-service/blob/master/docs/nydus-design.md